mantle/platform/conf/conf.go

```diff
@@ -398,6 +398,40 @@ func MergeAllV34exp(confObjs []*Conf) (*UserData, error) {
 	return userData, nil
 }
 
+// Config is compressed and added to another via data url
+func ReplaceAndCompress(config string) (*Conf, error) {
```

Comment:
I wonder if we should return the config also as a string so it's easy for callers to use as an optional "filter". Would probably need to do this anyway to do the length check as suggested in the previous comment.

Reply:
I decided to return `*Conf` because I thought it was more aligned with the other functions in that file. In the case of more config complex manipulations, a `*Conf` return value is probably more useful, but I agree that just for our specific use here we can return a string.